composer.json

```diff
@@ -10,11 +10,11 @@
         }
     ],
     "require": {
-        "php": ">= 7.1",
+        "php": ">=7.1",
         "phpdocumentor/reflection-docblock": "^3.2.3|^4.0.1",
         "tebru/php-type": "^0.1.7",
-        "tebru/doctrine-annotation-reader": "^0.3.6",
-        "symfony/cache": "^3.3|^4.0",
+        "tebru/doctrine-annotation-reader": "^0.3.7",
+        "symfony/cache": ">=4.0 <4.3",
```

Comment:
I still need to support symfony 3

Reply:
where did 5 go?

Reply:
I commited a test sorry :confused:

Reply:
@natebrunette finally fixed... sorry about the mess